BrainPortal/cbrain_plugins/cbrain-plugins-base/userfiles/mp3_audio_file/views/_html5_mp3_audio.html.erb

```diff
@@ -25,7 +25,7 @@
 <p>Note: for the moment, this viewer code only works reliably in Google Chrome.</p>
 
 <audio controls="controls" preload="none">
-  <source src="<%= content_userfile_path(@userfile) %>" type="audio/mp3" />
+  <source src="<%= content_userfile_path(@userfile, :content_loader => :collection_file, :arguments => @userfile.name) %>" type="audio/mp3" />
```

Comment:
This means you are providing a content_loader named "collection_file" even when the userfile is a standlone single file, which don't have any content loader of that name.